lasagne/tests/layers/test_special.py

```diff
@@ -17,12 +17,12 @@ def layer_vars(self, NonlinearityLayer, dummy_input_layer):
         layer = NonlinearityLayer(
             dummy_input_layer,
             nonlinearity=nonlinearity,
-            )
+        )
```

Comment:
Not your business! :) Please don't commit any whitespace changes in unrelated code. (Some editors do that automatically, reconfigure your editor or just make sure not to commit such changes.)